.env.example

```diff
@@ -1,3 +1,4 @@
+SIGNAL_VERSION="v0.0.1"
```

Comment:
Could you load this from the `package.json` file instead when generating the script?

Reply:
I was thinking about that too... Ultimately I've decided on this approach thinking about "custom versioning" when people are building their own script.

Do you think it's clear enough that if they want to ship a custom version (let's say they want to do `v1.0.0-shop`), they need to change the package.json themselves?

Reply:
Hmm, I hadn't thought of it this way yet.

What would differentiate a custom version from the Signal version? Generating with different settings?

Reply:
Would you drop the `v` prefix here? It's not actually part of the version number and doesn't convey useful information if you think about it in a DB column. WDYT?

Reply:
I'd also like to understand what you mean by custom version a bit better. Would that be a fork of Signal? At that point this would effectively be a version reset, no?